packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -50,6 +51,8 @@ const PipelineRuns = ({
   },
   getPipelineRunCreatedTime = pipelineRun =>
     pipelineRun.metadata.creationTimestamp,
+  getPipelineRunId = pipelineRun =>
+    `${pipelineRun.metadata.namespace}:${pipelineRun.metadata.name}`,
```

Comment:
since we're changing this anyway can we just make it `pipelineRun.metadata.uid`?

Reply:
I was going to but for the sake of our limbo state I decided against. uid is generated by k8s after the resource has been applied. namespace and name are expected even for pipelineRuns that have not been. In practice it should never break the dashboard but this is the safer option imo

Reply:
fair enough